tensorflow/c/c_api.h

```diff
@@ -1662,6 +1662,29 @@ TF_CAPI_EXPORT extern TF_Buffer* TF_GetAllRegisteredKernels(TF_Status* status);
 TF_CAPI_EXPORT extern TF_Buffer* TF_GetRegisteredKernelsForOp(
     const char* name, TF_Status* status);
 
+// --------------------------------------------------------------------------
+// Server functionality.
+
+// Server.
```

Comment:
Adding some detail in the comments will be useful, as this is the only form of documentation we have for the C API. Same goes for all other functions added.